.github/workflows/CI.yml

```diff
@@ -81,6 +81,54 @@ jobs:
         run: |
           ./gradlew publishToMavenLocal -Dopensearch.version=1.0.0-rc1
 
+      - name: Multi Nodes Integration Testing
+        run: |
+          ./gradlew integTest  -PnumNodes=3
+
+      - name: Pull and Run Docker
+        run: |
+          ## plugin=`ls build/distributions/*.zip`
+          ## version=`echo $plugin|awk -F- '{print $4}'| cut -d. -f 1-3`
+          ## plugin_version=`echo $plugin|awk -F- '{print $4}'| cut -d. -f 1-4`
+          ## TODO: remove these two hard code versions below after GA release
+          version=1.0.0-rc1
+          plugin_version=1.0.0.0-rc1
+          echo $version
+          cd ..
+          if docker pull opensearchstaging/opensearch:$version
+          then
+            echo "FROM opensearchstaging/opensearch:$version" >> Dockerfile
+            ## The ESRestTest Client uses http by default.
+            ## Need to disable the security plugin to call the rest api over http.
```

Comment:
The comment does not apply  to the code.

Reply:
removed